### PR TITLE
GC Features: JS equalheight to CSS equalheight

### PR DESCRIPTION
**Pull request by Garneauma**
Changing GC Features component to use CSS equalheight feature instead of its JavaScript equivalent.

Related to WET-269

### CODE REVIEW

File: components/gc-features/_base.scss [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 	.well {
 		border-radius: 0;
+		height: 100%;
```
Comment (duboisp) [inline components/gc-features/_base.scss:25]: This need to be tested: Text-zoom. But we should not need to use any "height:100%".

More rational will be needed to understand if this don't reveal a bug with the equal height CSS feature we have.

File: components/gc-features/gc-features-en.html [hunk 2]
```diff
@@ -41,23 +41,23 @@ <h2>Working example</h2>
 <h3>Context-specific features - v9.5.0</h3>
 <section class="gc-features">
 	<h2>Features</h2>
-	<div class="row wb-eqht">
+	<div class="row wb-eqht-grd">
```
Comment (duboisp) [inline components/gc-features/gc-features-en.html:44]: The old/deprecated pattern need to be documented (working example)

**Review by duboisp — CHANGES_REQUESTED** [review]
The old/deprecated pattern need to be documented (working example)

File: components/gc-features/_base.scss (hunk 1 above)
Comment (Garneauma) [reply components/gc-features/_base.scss:25]: Removed `height: 100%` and reinstated `.eqht-trgt`

**Review by duboisp — CHANGES_REQUESTED** [review]
The component version need to be majored to v5.0.

Mention that Version 4.0 are still valid and supported.

**Comment by duboisp** [comment]
Approved once local testing completed and the documentation change about the component version is completed.

File: components/gc-features/gc-features-en.html [hunk 3]
```diff
@@ -33,20 +33,20 @@
 	</dd>
 
 	<dt>Component version</dt>
-	<dd>Version 4.0</dd>
+	<dd>Version 5.0</dd>
```
Comment (duboisp) [inline components/gc-features/gc-features-en.html:36]:
```suggestion
	<dd>Version 5.0</dd>
	<dt>Supported working version</dt>
	<dd>
		<ul>
			<li>Version 5</li>
			<li>Version 4</li>
			<li>Version 3 (deprecated)</li>
			<li>Version 2 (deprecated)</li>
			<li>Version 1 (deprecated)</li>
		</ul>
	</dd>
```

**Review by duboisp — CHANGES_REQUESTED** [review]
Just one small thing, let's specify we support version 4 and version 5. And that older version are deprecated. That way we will be able to mark this change as a minor change.

I tested locally and work as expected. Once the little content change we will merge it.

File: components/gc-features/gc-features-fr.html [hunk 4]
```diff
@@ -33,7 +33,18 @@
 	</dd>
 
 	<dt>Component version</dt>
-	<dd>Version 4.0</dd>
+	<dd>Version 5.0</dd>
+
+	<dt>Supported working versions</dt>
```
Comment (duboisp) [inline components/gc-features/gc-features-fr.html:38]:
```suggestion
	<dt>Version fonctionnelle compatible</dt>
```

**Review by duboisp — CHANGES_REQUESTED** [review]

File: components/gc-features/gc-features-fr.html [hunk 5]
```diff
@@ -33,7 +33,18 @@
...
+	<dd>
+		<ul>
+			<li>Version 5</li>
+			<li>Version 4</li>
+			<li>Version 3 (deprecated)</li>
+			<li>Version 2 (deprecated)</li>
+			<li>Version 1 (deprecated)</li>
```
Comment (duboisp) [inline components/gc-features/gc-features-fr.html:45]:
```suggestion
	<dt>Version  compatible et fonctionnelle</dt>
	<dd>
		<ul>
			<li>Version 5</li>
			<li>Version 4</li>
			<li>Version 3 (désuet)</li>
			<li>Version 2 (désuet)</li>
			<li>Version 1 (désuet)</li>
```

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — APPROVED** [review]

**Comment by duboisp** [comment]
This is a minor change

**Pull request merged**